test/e2e/etcd_operator_unannotated_test.go

```diff
@@ -0,0 +1,78 @@
+package e2e
+
+import (
+	"context"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+	"time"
+)
+
+func ServiceBindingRequestSetup(t *testing.T, steps []Step) {
+	t.Log("Creating a new test context...")
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup()
+
+	ns, f := bootstrapNamespace(t, ctx)
+
+	// executing testing steps on operator
+	serviceBindingRequestTestSetup(t, ctx, f, ns, steps)
+}
+
+func serviceBindingRequestTestSetup(t *testing.T, ctx *framework.TestCtx, f *framework.Framework, ns string, steps []Step) {
```

Comment:
Please leave a documentation comment explaining what this step is doing, and what's expected from it. Also, the line seems to be way over 101 columns limit (`.editorconfig` settings).

Reply:
Outdated